modules/configuration/js/configuration_helper.js

```diff
@@ -86,6 +86,9 @@ $(function () {
             error: function(xhr, desc, err) {
                 console.log(xhr);
                 console.log("Details: " + desc + "\nError:" + err);
+                var html = "<label>Duplicate Values</label>";
```

Comment:
Could maybe make the error more explicit i.e. 'One or more fields with multiple-entry option has duplicate values'